test/rhsmlib/dbus/test_register.py

```diff
@@ -475,3 +503,19 @@ def test_RegisterWithActivationKeys(self):
             self.LOCALE,
         )
         self.assertEqual(expected, json.loads(result))
+
+    def test_RegisterWithActivationKeys__with_force_option(self):
+        expected = json.loads(CONSUMER_CONTENT_JSON)
+        self.patches["is_registered"].return_value = False
+        self.patches["unregister"].return_value = None
+        self.patches["register"].return_value = expected
+
+        result = self.obj.RegisterWithActivationKeys.__wrapped__(
+            self.obj,
+            "username",
+            ["key1", "key2"],
+            {"force": True},
+            {"host": "localhost", "port": "8443", "handler": "/candlepin"},
+            self.LOCALE,
+        )
+        self.assertEqual(expected, json.loads(result))
```

Comment:
Maybe it would be good to add unit test `test_RegisterWithActivationKeys__already_registered` too.